tests/butlerAlias.py

```diff
@@ -65,7 +65,7 @@ def getDefaultLevel(self):
         return 'visit'
 
     def getKeys(self, datasetType, level):
-        return {'filter': types.StringType, 'visit': types.IntType}
+        return {'filter': bytes, 'visit': int}
```

Comment:
Thank you. With so many changes it's easy to miss some. futurize knows that `types.StringType` is a synonym for `bytes` and since `StringType` does not exist on python3 it just buts `bytes` in there.